index.html

```diff
@@ -114,7 +114,33 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-        
+      
+       <!-- ________ Princess's card START ________  -->
+        <div class="card">
+          <p class="name">Princess Enuma</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">@alwayspearlxx</a>
+          </p>
+          <p class="about">I'm a Front-end developer, I enjoy coding and i love listening to music and watching anime.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">w3Schools</a>
```

Comment:
enter the link to those website you think as useful resoureces  and not just '#" and net naija is a movie download websites, doesn't contain any coding tools.